public/app/pages/configuration.js

```diff
@@ -393,6 +404,9 @@ Vue.component('Configuration', {
 				weather: this.weather,
 				track: this.track.name,
 				track_config: this.track.config,
+				legal_tyres: this.legal_tyres,
+				udp_plugin_local_port: this.udp_plugin_local_port,
```

Comment:
Add parseInt() for numbers: udp_plugin_local_port: parseInt(this.udp_plugin_local_port),